09-redux/classworks/Gloria_Loi/src/pages/Routes.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Switch, Route, Redirect } from 'react-router-dom';
+
+import useRedditApi from 'hooks/useRedditApi';
+
+import Login from 'pages/login';
+import Callback from 'pages/callback';
+import Home from 'pages/home';
+
+const Routes = (props) => {
+  const [, , isLoggedIn] = useRedditApi();
+
+  if (isLoggedIn) {
+    return (
+      <Switch>
+        <Route path='/' render={() => <Home />} />
```

Comment:
As we don't pass props to Home from here anymore we can just pass Home component in Route (see examples below)